pkg/core/native/util.go

```diff
@@ -64,6 +70,20 @@ func setUint32WithKey(id int32, dao dao.DAO, key []byte, value uint32) error {
 	return dao.PutStorageItem(id, key, si)
 }
 
+func setBigIntWithKey(id int32, dao dao.DAO, key []byte, value *big.Int) error {
```

Comment:
Maybe `setIntWithKey(..., value int64) error` would be easier to handle? I don't think we ever go beyond int64 and `bigint.*` conversions can be handled internally.